src/app/header/header.component.scss

```diff
@@ -1,31 +1,30 @@
-header {
-  background-color: var(--ds-header-bg);
-}
+:host {
+  header {
+    background-color: var(--ds-header-bg);
+  }
 
-.navbar-brand img {
-  max-height: var(--ds-header-logo-height);
-  max-width: 100%;
-  @media screen and (max-width: map-get($grid-breakpoints, sm)) {
-    max-height: var(--ds-header-logo-height-xs);
+  .navbar-brand img {
+    max-height: var(--ds-header-logo-height);
+    max-width: 100%;
   }
-}
 
-.navbar-toggler .navbar-toggler-icon {
-  background-image: none !important;
-  line-height: 1.5;
-}
+  .navbar-toggler .navbar-toggler-icon {
+    background-image: none !important;
+    line-height: 1.5;
```

Comment:
This should be (derived from) a variable

Reply:
See comment above

https://github.com/DSpace/dspace-angular/pull/2676#discussion_r1475977391